app/models/discussion.rb

```diff
@@ -19,7 +22,7 @@ def date_during_event_duration
   end
 
   def end_time_greater_than_start_time
-    errors[:end_time] << "should be more than start time" if start_time > end_time
+    errors[:end_time] << "should be more than start time" if start_time >= end_time
   end
 
 
```

Comment:
please see to upcoming? method
line number 33
you are doing something like `return true if true` and `return false if true`